apps/camera/js/views/controls.js

```diff
@@ -251,10 +258,10 @@ module.exports = View.extend({
     '</div>' +
     '<div class="controls-right">' +
       '<div class="mode-switch test-switch" name="switch">' +
-        '<div class="inner js-switch">' +
-          '<div class="mode-switch_bg-icon rotates" data-icon="camera"></div>' +
-          '<div class="mode-switch_bg-icon rotates" data-icon="video"></div>' +
-          '<div class="mode-switch_handle js-switch-handle">' +
+        '<div class="inner js-switch" role="radiogroup">' +
+          '<div class="mode-switch_bg-icon js-radio-camera rotates" data-icon="camera" role="radio" aria-checked="true" data-icon="camera" data-l10n-id="camera-mode-radio-button" aria-label="Camera mode"></div>' +
+          '<div class="mode-switch_bg-icon js-radio-video rotates" data-icon="video" role="radio" aria-checked="false" data-icon="video" data-l10n-id="video-mode-radio-button" aria-label="Video mode"></div>' +
```

Comment:
Why do we need the `aria-checked` and `aria-label` attributes in the markup? Won't they both get added automatically by l10n.js and `updateSwitchPosition()`?